green/output.py

```diff
@@ -167,6 +171,9 @@ def writeln(self, text=''):
     def write(self, text):
         if type(text) == bytes:
             text = text.decode('utf-8')
+        # Compensate for windows' anti-social unicode behavior
+        if self._ascii_only_output:
+            text = unidecode(text)
```

Comment:
Hmm, wait.  Python 3 doesn't have the unicode() function anymore.

Since it's a Python 2-specific fix, I suppose we should make that condition:

    if self._ascii_only_output:
        if platform.python_version_tuple()[0] == '2': # pragma: no cover
           text = unicode(text)
        text = unidecode(text)